### PR TITLE
Update blog

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2022/11/scaling-node-red-with-diy-tooling.md [hunk 1]
```diff
@@ -86,7 +86,9 @@ Many other edge cases itch in the back of our heads, but we can't focus on those
 Node-RED is a great tool, it's got many built-in features that make it easy to
 get started with no coding experience necessary. Running it at scale, in a
 production environment can require a lot of sys-ops and dev-ops time and we
-think FlowFuse is a great solution to keep that admin and tech debt in check.
+think [FlowFuse](/) is a great solution to keep that admin and tech debt in check.
 
 If you're running into these challenges we believe we can help, you can adopt
 our [free and open source edition](/docs/install/).
+
+Additionally, to get a head start or enhance your current setup, explore our [Beginner's Guide to Professional Node-RED](/ebooks/beginner-guide-to-a-professional-nodered/). This concise ebook offers a clear overview of Node-RED’s capabilities and practical tips for making the most of its features.
```
Comment (Hasmin-AC) [inline src/blog/2022/11/scaling-node-red-with-diy-tooling.md:94]:
```suggestion
Additionally, to get a head start or enhance your current setup, explore our [Beginner's Guide to Professional Node-RED](/ebooks/beginner-guide-to-a-professional-nodered/). This comprehensive ebook offers a clear overview of Node-RED’s capabilities and practical tips for making the most of its features.
```

**Review by Hasmin-AC — APPROVED** [review]
